client/src/components/app.jsx

```diff
@@ -1,33 +1,51 @@
 const React = require('react');
 const $ = require('jquery');
+const { Deals } = require('./deals.jsx');
```

Comment:
I would suggest using the ES6 module imports . Here's an example:
```javascript
import React from 'react';
import $ from 'jquery';
import { Deals } from './deals';
```

Reply:
Did a little research and will change all of the require/module.exports to import/export. Thank you for the suggestion!

Reply:
The change will be included in the next PR